File: python/src/tests/t_statistics.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

result = atrshmlog.attach()

maxindex = atrshmlog.get_statistics_max_index()

print('statistics : ' + str(maxindex) + ' : ')

stats = atrshmlog.get_statistics()

for i in range(0, maxindex) :
    print('statistics : ' + str(i) + ' : ' + str(stats[i]) + ' : ')

print (' ')

exit(0)

# end of test

File: perl/src/tests/t_statistics.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$maxindex = Atrshmlog::get_statistics_max_index();

print "statistics : $maxindex : \n";

@stats = Atrshmlog::get_statistics();

print "statistics : $#stats : \n";

$index = 0;
if ($#stats > 0) {
    foreach $s (@stats) {
	print "statistics : $index : $s : \n";
	++$index;
    }
}

print " \n";

exit (0);

# end of main
